### DISCUSSION TITLE
[第1期组队学习] 队友征集帖

### DISCUSSION THREAD

**Discussion by anine09** [datawhalechina/learn-python-the-smart-way-v2#11] [P2S 妙妙屋]
# 召集队友，一块学习、一块成长、一块快乐！

大家可以在这个帖子里直接回复，然后盖楼。勾搭大佬，找到组队的小伙伴。

如果大家组队成功，可以自己先小队建群，推选队长，队长可参与“优秀队长”的评选（颁发 Datawhale 学习证书）。

队长的职责一般是：

- 活跃小队同学们的学习积极性
- 将大家的困难反馈到教学团队
- 小队中的精神旗帜

**一旦推选出队长，请队长联系[教学团队](https://datawhalechina.github.io/learn-python-the-smart-way-v2/Team/team/)，告知你的联系方式，教学团队会邀请队长进入本次的微信群。**

关于组队一般有一些小方法，可以参照以下模版：

```
大家好，我是...
【坐标】：天津
【专业/职业】：软件工程研0
【感兴趣的方向】：自然语言处理
【是否愿意作队长】：是
【兴趣爱好】：运动，阅读
【目标】：掌握深度学习
```

很多同学对自己的能力不自信，不敢当队长。放心，哪位大佬不是从小白过来的呢？这是个成长的好机会，大胆一点。

任何问题都可以联系[教学团队](https://datawhalechina.github.io/learn-python-the-smart-way-v2/Team/team/)求助。

**Comment by anine09** [comment]
（示例，无法与教学团队组队👻）
大家好，我是骆秀韬
【坐标】：武汉
【专业/职业】：材料物理大二
【感兴趣的方向】：自然语言处理，AI for Science
【是否愿意作队长】：是
【兴趣爱好】：爱好男高
【目标】：带领大家冲向人工智能训练大师的高峰！

**Reply by anine09** [reply]
大家可以直接在下方评论，跟发布人联系

**Reply by anine09** [reply]
如果组队人数超过三人 就可以向教学团队汇报了

**Reply by 94jia** [reply]
可以组队吗

**Comment by bunny-sadako** [comment]
大家好，我是徐铁真
【坐标】：邯郸
【专业/职业】：测控研一
【感兴趣的方向】：医学图像分割
【是否愿意作队长】：是
【兴趣爱好】：天文地理，诗词歌赋，魔方
【目标】：发SCI
微信：13940263058
欢迎大家一起组队！

**Reply by Nengzyue** [reply]
【坐标】：杭州
【专业/职业】：交通研一
【感兴趣的方向】：自然语言处理、大数据
【是否愿意作队长】：是
【兴趣爱好】：运动，唱跳和Rap
【目标】：入门Python基础学习

**Comment by zarjun** [comment]
大家好，我是张不懂
【坐标】：上海
【专业/职业】：电科研一
【感兴趣的方向】：自然语言处理
【是否愿意作队长】：是
【兴趣爱好】：运动，唱跳和Rap
【目标】：入门Python基础学习

**Reply by harduz** [reply]
你好，可组队吗？
大家好，我是阿白
【坐标】：江苏
【专业/职业】：工程师
【感兴趣的方向】：大语言模型
【兴趣爱好】：器乐
【目标】：入门Python基础学习

**Comment by yookbu** [comment]
大家好，我是yookbu
【坐标】：宁夏
【专业/职业】：前端开发
【感兴趣的方向】：图像分类、图像处理
【是否愿意作队长】：是
【兴趣爱好】：运动
【目标】：入门Python基础学习

**Comment by hu-qi** [comment]
大家好，我是 huqi
【坐标】：广州
【专业/职业】：切图仔
【感兴趣的方向】：CV
【是否愿意作队长】：是
【兴趣爱好】：Coding
【目标】：巩固 Python 基础，掌握深~~夜~~度学习

**Reply by anine09** [reply]
深夜学习可还行

**Comment by 13175802179** [comment]
大家好，我是
【坐标】：合肥
【专业/职业】：物流研一
【感兴趣的方向】：数据挖掘、遥感交通碳排放预测
【是否愿意作队长】：是
【兴趣爱好】：骑行、游泳、健身
【目标】：发SCI
微信：13175802179
欢迎大家一起组队！

**Reply by zhenbengbuzhule** [reply]
你好，可组队吗？
大家好，我是Leo
【坐标】：上海
【专业/职业】：物流本科在读
【感兴趣的方向】：机器学习
【兴趣爱好】：运动
【目标】：入门Python基础学习
微信：15001829365

**Comment by zyl9737** [comment]
大家好，我是Yulong
【坐标】：深圳
【专业/职业】：电信研0
【感兴趣的方向】：CV，安全
【是否愿意作队长】：是
【兴趣爱好】：运动，阅读
【目标】：巩固Python，参与开源
微信：zyl9737

**Reply by maruizhe** [reply]
你好，本人哈工大自动化大二，愿意一同组队，同对CV和安全方面感兴趣

**Reply by anine09** [reply]
赞 三人组队就可以联系助教了

**Reply by LAYCLOUDS** [reply]
还缺人吗？

**Reply by zyl9737** [reply]
可以直接加我微信哦

**Reply by rbpi** [reply]
这个感兴趣的方向中，提及的安全是指信息安全吗？

**Reply by zyl9737** [reply]
主要的方向是图像、文本之类的伪造检测，不是渗透和web安全那一块噢（只是学习这个课程的话，感兴趣的方向不重要，想学Python就欢迎加入o：）

**Reply by Hallock** [reply]
加入

**Reply by MYD753Q** [reply]
加入

**Comment by bunny-sadako** [comment]
你好，可以的，加我微信13940263058



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年7月20日(周四) 晚上10:08
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [datawhalechina/learn-python-the-smart-way-v2] [第1期组队学习] 队友征集帖 (Discussion #11)




 
你好，可组队吗？
 大家好，我是阿白
 【坐标】：江苏
 【专业/职业】：工程师
 【感兴趣的方向】：大语言模型
 【兴趣爱好】：器乐
 【目标】：入门Python基础学习

**Comment by bunny-sadako** [comment]
加我微信聊13940263058



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年7月20日(周四) 凌晨0:09
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [datawhalechina/learn-python-the-smart-way-v2] [第1期组队学习] 队友征集帖 (Discussion #11)




 
你好，本人哈工大自动化大二，愿意一同组队，同对CV和安全方面感兴趣

**Comment by bunny-sadako** [comment]
可以，组队加我微信13940263058



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年7月22日(周六) 晚上9:25
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [datawhalechina/learn-python-the-smart-way-v2] [第1期组队学习] 队友征集帖 (Discussion #11)




 
你好，可组队吗？
 大家好，我是Leo
 【坐标】：上海
 【专业/职业】：物流本科在读
 【感兴趣的方向】：机器学习
 【兴趣爱好】：运动
 【目标】：入门Python基础学习
 微信：15001829365

**Comment by 13175802179** [comment]
可以












张梓淇
***@***.***


学生
13175802179


---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年7月23日(周日) 晚上8:24
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [datawhalechina/learn-python-the-smart-way-v2] [第1期组队学习] 队友征集帖 (Discussion #11)




 
可以组队吗

**Comment by bunny-sadako** [comment]
可以的，加我微信就好13940263058



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年7月23日(周日) 晚上8:24
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [datawhalechina/learn-python-the-smart-way-v2] [第1期组队学习] 队友征集帖 (Discussion #11)




 
可以组队吗

**Reply by 94jia** [reply]
加了

**Comment by wly-115** [comment]
大家好，我是侦察矮人
【坐标】：陕西
【专业/职业】：研一
【感兴趣的方向】：量子计算
【是否愿意作队长】：都可以
【兴趣爱好】：运动，打游戏
【目标】：巩固Python，参与开源
微信：flashrun123

**Reply by MYD753Q** [reply]
组队滴滴

**Comment by ilcan12** [comment]
大家好，我是沐卿朝（VX）
【坐标】：杭州
【专业/职业】：计科大一结束（因转专业，语言这一块不太扎实）
【感兴趣的方向】：机器学习，计算机视觉
【是否愿意作队长】：都OK（不过本人很菜）
【兴趣爱好】：听音乐
【目标】：掌握py，根据爬取的数据制作喜欢的数据可视化图表

**Comment by bunny-sadako** [comment]
不缺了



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年7月28日(周五) 晚上9:32
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [datawhalechina/learn-python-the-smart-way-v2] [第1期组队学习] 队友征集帖 (Discussion #11)




 
还缺人吗？

**Comment by 13175802179** [comment]
现在六个人了












张梓淇
***@***.***


学生
13175802179


---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年7月28日(周五) 晚上9:39
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [datawhalechina/learn-python-the-smart-way-v2] [第1期组队学习] 队友征集帖 (Discussion #11)




 不缺了 
 
 
 
 ---原始邮件--- 
 发件人: ***@***.***&amp;gt; 
 发送时间: 2023年7月28日(周五) 晚上9:32 
 收件人: ***@***.***&amp;gt;; 
 抄送: ***@***.******@***.***&amp;gt;; 
 主题: Re: [datawhalechina/learn-python-the-smart-way-v2] [第1期组队学习] 队友征集帖 (Discussion #11) 
 
 
 
 
   
 还缺人吗？

**Reply by anine09** [reply]
满三个人就可以联系[教学团队](https://datawhalechina.github.io/learn-python-the-smart-way-v2/Team/team/)了哦

**Comment by LAYCLOUDS** [comment]
还有没有缺人的啊~~~

**Comment by LAYCLOUDS** [comment]
话说组不到队可以自己一个人吗

**Reply by anine09** [reply]
可以，个人当然可以学

**Comment by MYD753Q** [comment]
大家好，我是MYD
【坐标】：上海
【专业/职业】：AI 大四
【感兴趣的方向】：CV，自然语言处理，机器学习
【是否愿意作队长】：是
【兴趣爱好】：打球
【目标】：熟悉python语法，为后续做准备

**Reply by escape-free** [reply]
大佬可以一起组队吗

**Reply by MYD753Q** [reply]
忘记贴微信：MYD753Q

**Reply by Moon-creat** [reply]
还能组队吗
